src/main/frontend/components/FilmShow.js

```diff
@@ -100,17 +103,56 @@ const FilmShow = props => {
       />)
   }
 
+  const deleteFilm = async() => {
+    try {
+      const response = await fetch(`/api/v1/films/${props.match.params.id}/delete`, {
```

Comment:
I would use this as the path to follow RESTful patterns `/api/v1/films/${props.match.params.id}`.  Since this is handling DELETE requests being sent there will be no confusion with a GET request sent to the same path